shell/platform/android/test/io/flutter/embedding/android/FlutterActivityAndFragmentDelegateTest.java

```diff
@@ -451,9 +451,16 @@ public void itSendsMessageOverSystemChannelWhenToldToTrimMemory() {
     delegate.onAttach(RuntimeEnvironment.application);
```

Comment:
nit: you should change the test name now

Reply:
done